### PR TITLE
[utilities] Replace 'could' and 'might'

### PR DESCRIPTION
**Pull request by jensmaurer**
as directed by ISO/CS.

Partially addresses #4319

### CODE REVIEW

File: source/utilities.tex [hunk 1]
```diff
@@ -15121,8 +15122,8 @@
 
 \pnum
 \begin{note}
-The types deduced by the deduction guides for \tcode{function}
-might change in future revisions of \Cpp{}.
+It is possible that the types deduced by the deduction guides for \tcode{function}
+change in future revisions of \Cpp{}.
```
Comment (zygoloid) [inline source/utilities.tex:15126]: I don't think this has the same implications. @jwakely, what about: "The types deduced by the deduction guides for `function` have been identified as candidates for being changed in a future revision of \Cpp{}."? Is that accurate?

Comment (jwakely) [reply source/utilities.tex:15126]: Yes, I think so.

Comment (jensmaurer) [reply source/utilities.tex:15126]: fixed

File: source/utilities.tex [hunk 2]
```diff
@@ -4724,7 +4724,8 @@
 
 \pnum
 \begin{note}
-A \tcode{variant} might not hold a value if an exception is thrown during a
+It is possible for a \tcode{variant} not to hold a value
+if an exception is thrown during a
```
Comment (jwakely) [inline source/utilities.tex:4728]: Another awkward "possible not to ..." but I don't have a better suggestion.

**Review by jwakely — APPROVED** [review]

File: source/utilities.tex (hunk 2 above)
Comment (jwakely) [reply source/utilities.tex:4728]: Ah , in [variant.variant.general] we already say "or holds no value", so if we don't mind using "hold no value" as the opposite of "hold a value", we could say "It is possible for a `variant` to hold no value if ...".

Comment (jensmaurer) [reply source/utilities.tex:4728]: fixed

**Review by jwakely — APPROVED** [review]
Great, thanks!

**Pull request merged**